src/stories/components/MainMenu/styles.scss

```diff
@@ -0,0 +1,33 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Main Menu
+
+// .main-menu {}
+
+.menu-item {
+    min-width: 200px;
+    max-width: 200px;
```

Comment:
Not sure if you need to specify the min and max width since I'm pretty sure the component will be used with layout from Sidebar to create the page. Let's wait for other to chime in on this one. It might be okay for now.

Reply:
No need for this!

Reply:
I will remove it!